cmd/fyne/internal/templates/gen.go

```diff
@@ -0,0 +1,59 @@
+// +build ignore
```

Comment:
This is quite a bit of duplicated code, can we re-use the internal code or use the "fyne package" tool that has it built as an external app?

Reply:
For future reference: Using `go:genernate` works like a champ :)